eclipse-platform/eclipse.platform.ui
# Improve text for sticky scrolling options

The text for the maximum lines count setting should end with a colon, like the other options.
Before:
<img width="650" alt="Bildschirmfoto 2024-07-15 um 12 17 06" src="https://github.com/user-attachments/assets/6cbebccc-4e91-43c9-8aed-de5d1d9152e7">

After:
![image](https://github.com/user-attachments/assets/6f4314bb-d498-4419-be9d-c79eac2b7a84)

### Testing
1. Open the preferences
2. Navigate to "Text Editors"
3. Check the text for the sticky scrolling settings
